README.md

```diff
@@ -43,7 +43,7 @@ For more information about Gossamer, the Polkadot ecosystem, and how to use Goss
 
 ### Prerequisites
 
-install go version `>=1.16`
+install go version `>=1.17`
```

Comment:
I put 1.16 on purpose, I think it should work still.
We use `io` and `embed` from 1.16, however I realised our go.mod got 1.17-updated so maybe it doesn't work with 1.16 :thinking: They sort of broke compatibility with that go.mod change when I think about it...

Reply:
go 1.16 doesn't work anymore as we use `time.UnixMilli` which is only in go 1.17

Reply:
Ok so actually, the go.mod still works on 1.16 but we do use `time.UnixMilli` which is only here with 1.17. APPROVED!!